src/main/resources/site/parts/article/article.html

```diff
@@ -23,7 +23,7 @@ <h1 class="col-lg-10" data-th-text="${title}" />
             </div>
             <div class="author-wrapper">
                 <div class="row p-0" data-th-if="${#lists.size(authors) >= 1}">
-                    <p class="font-weight-bold mr-1" data-th-text="(${language.phrases.author} + ':')" />
+                    <p class="mr-1" data-th-text="(${language.phrases.author} + ':')" />
```

Comment:
Any reason why the term should not be in bold ? (the section looks very neat and complete with term names in bold and values normal-weight, doesn't it ?)

Reply:
The design https://app.zeplin.io/project/5d01fefdc297435df2566282/screen/5e70f9a3489481b3c50af72e shows with normal weight